opal/datatype/opal_datatype_copy.c

```diff
@@ -157,19 +117,16 @@ int32_t opal_datatype_copy_content_same_ddt(const opal_datatype_t *datatype, int
      */
     extent = (datatype->true_ub - datatype->true_lb) + (count - 1) * (datatype->ub - datatype->lb);
 
-    fct = non_overlap_copy_content_same_ddt;
-    SET_GPU_COPY_FCT(device_bufs, fct, non_overlap_gpu_copy_function);
+    fct = non_overlap_accelerator_copy_content_same_ddt;
```

Comment:
does this function handle non-device buffers as well now? Previously we conditionally set copy function to accelerator type.

Reply:
Yes